src/upgrades/230627-upgrade-v0.5.2-testnet.ts

```diff
@@ -0,0 +1,123 @@
+/// npx hardhat deploy --tags 230627UpgradeTestnet --network ronin-testnet
+
+/// This script does the following:
+/// - Set new enforcer for mainchain gateway
+
+/// Governor who proposes this proposal must manually vote it after running this script.
+
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { EXPLORER_URL, proxyCall, proxyInterface } from './upgradeUtils';
+import { VoteType } from '../script/proposal';
+import { RoninGatewayV2__factory } from '../types';
+import { generalRoninConf } from '../configs/config';
+import { network } from 'hardhat';
+
+const deploy = async ({ getNamedAccounts, deployments, ethers }: HardhatRuntimeEnvironment) => {
```

Comment:
Since this script is only for testnet please add some restriction to ensure no one runs this by mistake

Reply:
I planned to reuse it on mainnet too. But will add the restriction temporarily to avoid misuse.